tests/specs/ecosystem/amit/neon_serverless_driver/0_0_3.test

```diff
@@ -68,4 +68,12 @@ amit/neon-serverless-driver/0.0.3
 
 == FAST CHECK EMIT PASSED ==
 
-== TYPE CHECK PASSED ==
+== TYPE CHECK FAILED ==
+-- stdout --
+
+-- stderr --
+error: TS2834 [ERROR]: Relative import paths need explicit file extensions in ECMAScript imports when '--moduleResolution' is 'node16' or 'nodenext'. Consider adding an extension to the import path.
+export { DatabaseError } from "./pg-protocol";
```

Comment:
Bug.

Reply:
I'm going to fix this in a follow-up because it requires a deno_graph fix I think.